.github/workflows/ci.yaml

```diff
@@ -39,22 +39,31 @@ jobs:
           go-version: "1.17"
       - name: Build
         run: |
-          GOOS=linux GOARCH=arm64 go build -o $BINARY_NAME-arm64 main.go      
+          GOOS=linux GOARCH=arm64 go build -o $BINARY_NAME-arm64 main.go
           GOOS=linux GOARCH=amd64 go build -o $BINARY_NAME-amd64 main.go
 
       - uses: aws-actions/configure-aws-credentials@v1
         with:
           role-to-assume: ${{ secrets.GU_RIFF_RAFF_ROLE_ARN }}
           aws-region: eu-west-1
-      - uses: guardian/actions-riff-raff@main
+      - uses: guardian/actions-riff-raff@v1.0.0
```

Comment:
The [recommendation](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstepsuses) seems to prefer referencing the major version:

> Using the specific major action version allows you to receive critical fixes and security patches while still maintaining compatibility. It also assures that your workflow should still work.

Not sure how many releases we'd make to the actions-riff-raff action though...

Reply:
ah yeah, good point! Much better this way.